### PR TITLE
AG-12838 Update Locales

### PR DESCRIPTION
**Pull request by JamesSwinton**
https://ag-grid.atlassian.net/browse/AG-12838

### CODE REVIEW

File: packages/ag-charts-locale/src/ar-EG.ts [hunk 1]
```diff
@@ -1,16 +1,28 @@
+/**
+ * Please note:
+ *
+ * Translations are provided as an illustration only and are
+ * not guaranteed to be accurate or error-free.
+ *
+ * They are designed to show developers where to store t heir
```
Comment (alantreadway) [inline packages/ag-charts-locale/src/ar-EG.ts]: Nit: `t heir` here and below?

Comment (JamesSwinton) [reply packages/ag-charts-locale/src/ar-EG.ts]: Good catch, thanks. fixed.

**Review by alantreadway — APPROVED** [review]
LGTM, merge on green!

**Comment by olegat** [comment]
FYI - There's two more strings incoming https://github.com/ag-grid/ag-charts/pull/2479.

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**